docs/quick-start/index.mdx

```diff
@@ -124,7 +124,7 @@ You can find more examples showcasing various features of Crawlee in the [Exampl
 
 **Related links**
 
-- [Environment variables](./guides/environment-variables)
```

Comment:
we should setup a redirect for the old URL

https://github.com/apify/crawlee/blob/master/website/docusaurus.config.js#L101

Reply:
Added - to `/docs/guides/configuration#environment-variables`

Reply:
oh, you can't really redirect to `/docs/guides/configuration#environment-variables` :/ `/docs/guides/configuration` it is 🤷‍♂️

Reply:
you would have to use different approach for that, its possible but not worth it here iguess